### ISSUE TITLE
mercurius-codegen generates wrong types for netsted lists

### ISSUE THREAD

**Issue by SkeLLLa** [mercurius-js/mercurius-typescript#8]
Consider there's a schema like
```graphql
type Foo {
  bars: [[Int]]
}
type Query {
  getFoo: Foo
}
```

Actual behaviour:
And codegen generates following type for bars:
```ts
Foo?: {
    bars?: LoaderResolver<Maybe<Array<[Int]>>, Foo, {}, TContext>;
}
```
which leads to ts error: `Property 'Foo' of exported interface has or is using private name 'Int'.`

Expected behaviour: 
```ts
Foo?: {
    bars?: LoaderResolver<Maybe<Array<[Scalars['Int']]>>, Foo, {}, TContext>;
}
```

**Comment by PabloSzx** [comment]
I'm not sure if your schema is not following spec, GraphQL doesn't support tupples: https://github.com/graphql/graphql-spec/issues/423#issuecomment-371633947

**Comment by PabloSzx** [comment]
If it simply was a type in your schema, I will need a small reproduction repo: https://stackoverflow.com/help/minimal-reproducible-example

**Comment by PabloSzx** [comment]
fixed in **mercurius-codegen@1.3.7**, thanks for reporting this issue @SkeLLLa
